src/components/Nav.js

```diff
@@ -0,0 +1,57 @@
+import React, { forwardRef } from 'react';
+import { Flex, Link, Box, List, ListItem } from '@chakra-ui/core';
```

Comment:
`Box` is unused here -- think this was just missed from initial

Reply:
@domitriusclark Removed